Proto/cg.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
```

Comment:
I wonder if this batch file is helpful - perhaps just calling crossgen directly in each case would be better.  Adding batch files to encapsulate process invocations tends to just obscure things another layer?